use video duration from events

muhammad-ammar opened the pull request:
[EDUCATOR-1411](https://openedx.atlassian.net/browse/EDUCATOR-1411)

brianhw commented on edx/analytics/tasks/insights/video.py:
```diff
@@ -326,18 +328,17 @@ def reducer(self, key, events):
                 old_time = float(old_time)
 
-            def start_viewing():
+            def start_viewing(video_duration):
```
I don't think this is needed -- video_duration is defined now on 331, so it's already in context.  For example, youtube_id is not passed into start_viewing(), nor is current_time and a raft of others.

brianhw commented on edx/analytics/tasks/insights/video.py:
```diff
@@ -408,7 +409,7 @@ def end_viewing(end_time):
                     ## self.incr_counter(self.counter_category_name, 'Discard Viewing', 1)
                     pass
-                viewing = start_viewing()
+                viewing = start_viewing(video_duration)
```
Again, I don't think this change is needed.

muhammad-ammar commented:
@brianhw Addressed your feedback. Video acceptance test is fixed and [build is passing](http://jenkins-ci.analytics.edx.org/view/ad-hoc/job/edx-analytics-pipeline-acceptance-test-manual/1415/console). Is this good to merge?

muhammad-ammar commented:
@edx/analytics This needs your attention. Please have a look.

muhammad-ammar commented:
@HassanJaveed84 Thanks a lot for all the help
